src/ibek/gen_scripts.py

```diff
@@ -28,41 +28,60 @@ def ioc_deserialize(ioc_instance_yaml: Path, definition_yaml: List[Path]) -> IOC
 
     Returns an in memory object graph of the resulting ioc instance
     """
+    all_values: Dict[str, str] = {}
+
     # Read and load the support module definitions
     for yaml in definition_yaml:
         support = Support.deserialize(YAML(typ="safe").load(yaml))
+        for definition in support.defs:
+            for value in definition.values:
+                all_values[value.name] = value.value
         make_entity_classes(support)
+        for definition in support.defs:
+            make_entity_context(definition)
 
     # Create an IOC instance from it
-    return IOC.deserialize(YAML(typ="safe").load(ioc_instance_yaml))
+    ioc_instance = IOC.deserialize(YAML(typ="safe").load(ioc_instance_yaml))
+    return ioc_instance
+
+
+def make_entity_context(definition: Definition):
+    """
+    Create a context dictionary for the given `Entity` instance
+    This is for use in Jinja expansion of instances of this Entity
+    """
+    context = asdict(definition)
+
+    for value in definition.values:
+        context[value.name] = value.value
+
+    setattr(definition, "__context__", context)
```

Comment:
or this

Reply:
Right that was just a horrible thing I forgot to revert. ooops.